src/components/countries/country/country.scss

```
@@ -0,0 +1,47 @@
+.country {
+  background: linear-gradient(
+    180deg,
+    rgba(255, 228, 133, 0.5) 21.88%,
+    rgba(186, 123, 0, 0.5) 100%
+  );
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.2);
+  border-radius: 15px;
+  margin-bottom: 2%;
+  color: #313131;
+  font-family: Roboto;
+  font-style: normal;
+  
+  &__content {
+    display: flex;
+    align-items: center;
+    padding: 2% 2% 2% 2%;

```

```suggestion
    padding: 2%;
```
In addition, any necessity to use `%` instead of pixels?